README.md

```diff
@@ -11,10 +11,20 @@ To build the SDK and see changes, follow these steps:
    git clone <repository_url>
    cd <repository_directory>
 
-2. Install dependencies and setup modules using setup.py :
+2. Make sure you have python installed. You can download it from [here](https://www.python.org/downloads/). After that install setuptools using:
+     ```bash
+     pip install setuptools
```

Comment:
When I tried running it, additional pip installs were needed. Please review and update accordingly.

Reply:
I coudn't find any other dependency other than setuptools for running setup.py. However it could differ from one system to other.